lib/newman_conway.py

```diff
@@ -2,9 +2,20 @@
 
 # Time complexity: ?
 # Space Complexity: ?
-def newman_conway(num):
+def newman_conway(num, storage=None):
```

Comment:
👍  Well done, but it looks like you started to do this recursively and switched to a loop, so you don't need the optional parameter.